mitodl/mitxpro
# update eslint utils

#### What are the relevant tickets?
fixes https://github.com/mitodl/mitxpro/issues/1084

#### What's this PR do?
update eslint utils to 1.4.1
 
#### How should this be manually tested?
Nothing should break.
Tests should pass.

Note: This PR is inspired by https://github.com/mitodl/open-discussions/pull/2168
